docs/kyma-environment-broker/04-01-cls-architecture.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Cloud Logging Service Architecture
+type: Architecture
+---
+
+Cloud Logging Service (CLS) is a managed logging service for shipping logs from Runtimes. It uses the Elastic Stack (open source logging platform Elasticsearch, Logstash, Kibana) to store, parse, and visualize the application log data coming from Kubernetes applications.
+
+ Kyma environment broker provisions a CLS instance for a global account, creates a binding and passes the CLS credentials to the provisioner as SKR overrides so that fluent bit in the SKR can be properly configured to push the logs to the CLS instance. The architecture diagram below explains the complete process
+
+![CLS diagram](./assets/cls-arch.svg)
+
+1. CSI sends the request to KEB to provision a new KEB.
+2. If there is not CLS instance present for the global account, the it provisions a new instance for CLS for the global account.
```

Comment:
Who is "it"? KEB? CSI? The arrow goes from CSI to KEB, so I assume CSI provides it.

```suggestion
2. If there is no CLS instance present for the global account, then CSI provisions a new instance for CLS for the global account.
```